tests/parsers/thermo_fisher_nanodrop_eight/testdata/Thermo_NanoDrop_Eight_example03.json

```diff
@@ -27,9 +27,15 @@
                                 "value": 0.185,
                                 "unit": "mAU"
                             },
+                            "processed data aggregate document": {
+                                "processed data document": [
+                                    {
```

Comment:
Did you add these lines manually? If they were generated I would expect to see the "mass conceentration" lines move right an intent or two. I think I'm ok with this for showing exactly what changed, but if we leave like this it might be worth following up to regenerate to make indenting look better

(and same question for all other changes to test JSON here)

Reply:
@bdworth I did not add the lines manually. I generated the files with the parser, but unstaged the identifier lines so it was easier to see what had changed on the files. It seems like doing that broke the formatting somehow. I'll fix it.